workflow/sample_files/base-location-capetown-zaf.xml

```diff
@@ -45,10 +45,6 @@
         </BuildingConstruction>
       </BuildingSummary>
       <ClimateandRiskZones>
-        <ClimateZoneIECC>
-          <Year>2006</Year>
-          <ClimateZone>3A</ClimateZone>
-        </ClimateZoneIECC>
```

Comment:
Why did you remove this?

Reply:
Because that's not the correct climate zone for Capetown.

Reply:
So why not change it to the correct value?

Reply:
Is IECC climate zone defined for Capetown?

Reply:
This change is probably fine, I just didn't understand what prompted you to make it. We can go with it.